Formula/ninja.rb

```diff
@@ -20,9 +20,8 @@ class Ninja < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "5ec37aa6d5145de2f396926c75891b578883e2a078989969c011548cae2005cf"
   end
 
-  depends_on "python@3.9"
-
-  uses_from_macos "curl" => :test
```

Comment:
@iMichka where did you see the error that made you add this again? The test seems to pass on Linux even without this now.

Reply:
See https://github.com/Homebrew/homebrew-core/pull/84724

Reply:
I did see #84724, which is why I tagged you here. That PR answers what the error was, but it doesn't say anything about where that error came from.

Reply:
I don’t know. I saw an error, I fixed it. I thought it was the right thing to do. I did not investigate the root cause. It just looked like a dependency was missing, so I added it, and the error was gone. Maybe the fix was the wrong one, and if we have a better solution, please go ahead 😉

Reply:
I imagine the issue might happen if curl is installed in the CI but a dependency of curl is uninstalled (homebrew-test-bot is not designed really to rely on formulae like that - hence our issues with using `HOMEBREW_FORCE_BREWED_CURL`).